core/chains/evm/config/toml/defaults.go

```diff
@@ -65,6 +74,64 @@ func init() {
 	slices.SortFunc(DefaultIDs, func(a, b *big.Big) int {
 		return a.Cmp(b)
 	})
+
+	// read the custom defaults overrides
+	dir := os.Getenv(string(env.CustomDefaultsEnvKey))
+	if dir == "" {
+		// short-circuit; no default overrides provided
+		return
+	}
+
+	// Read directory contents
+	entries, err := os.ReadDir(dir)
+	if err != nil {
+		fmt.Println("Error reading directory:", err)
+		return
+	}
+
+	for _, entry := range entries {
+		if entry.IsDir() {
+			// Skip directories
+			continue
+		}
+
+		path := dir + "/" + entry.Name()
+		file, err := os.Open(path)
+		if err != nil {
+			fmt.Println("Error opening file:", err)
+			continue
+		}
+
+		// Read file contents
+		b, err := io.ReadAll(file)
+		if err != nil {
+			fmt.Println("Error reading file:", err)
+			file.Close()
+			continue
+		}
+		file.Close()
+
+		var config = struct {
+			ChainID *big.Big
+			Chain
+		}{}
+
+		if err := cconfig.DecodeTOML(bytes.NewReader(b), &config); err != nil {
```

Comment:
Correct me if I'm wrong but these overrides are specified at runtime it seems? We're not statically embedding the overrides depending on a particular env var (seems hard / impossible now that I think about it since `go:embed` is on file level vars

Reply:
Does this imply that these defaults need to be manually shipped to nops separately if they're not embedded in the built binary?

Reply:
Yes but only if not using the official docker release which can bundle them in.

Reply:
I agree w/ @jmank88 - The easiest CX I can imagine for the NOPs is including the overrides directory and the env var in the docker image. That way they won't have to think about this at all. 

It will take some time before CCIP doesn't release its own image, so this approach seems the easiest from a release perspective as well to me.